content/Api/Blip.Api.Template.Facades/Blip.Api.Template.Facades.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
-    <ProjectGuid>sonarqube-guid-facades</ProjectGuid>
+    <ProjectGuid>{C4E5B837-8C8D-47ED-973E-FB70664FFB60}</ProjectGuid>
```

Comment:
whenever you build the project, or open in VS even, the placeholder gets replaced by the generated GUID. Revert back to the placeholder

Reply:
I forgot that point. Thanks for remember!